tests/test_study.py

```diff
@@ -437,3 +438,77 @@ def f(trial):
             assert df.params.y[i] == 2.5
             assert df.user_attrs.train_loss[i] == 3
             assert ('system_attrs', 'fail_reason') in df.columns
+
+
+@pytest.mark.parametrize('storage_mode', STORAGE_MODES)
+def test_create_study(storage_mode):
+    # type: (str) -> None
+
+    with StorageSupplier(storage_mode) as storage:
+        # Test creating a new study.
+        study = optuna.create_study(storage=storage, exist_ok=False)
+
+        if not isinstance(study.storage, optuna.storages.InMemoryStorage):
```

Comment:
Can we apply the early-return pattern here?
And I think the test case with `exist_ok=True` works with InMemoryStorage.

Reply:
You are right. I'll fix it.